src/locales/en/translation.json

```diff
@@ -789,7 +789,13 @@
     "walletHistory": "Wallet must be connected.",
     "loading": "Loading..."
   },
+  "validationErrors": {
+    "insufficientBalance": "insufficient funds available",
+    "minimumZero": "Minimum amount more than zero"
+  },
   "lendingPage": {
+    "deposit": "Deposit",
```

Comment:
```suggestion
    "deposit": "Lend",
```

Reply:
In my option make sense to rename the button to Lend because when clicked will have a popup with  button to Deposit.